README.md

```diff
@@ -15,16 +15,29 @@
 
 
 ## Prerequisites
-1. [Just command runner](https://github.com/casey/just)
+1. [Hermit by Cashapp](https://cashapp.github.io/hermit/usage/get-started/)
 2. Elasticsearch with the default username & password (`elastic` & `changeme`) running on the default port (`http://localhost:9200`)
 3. Kibana with running on the default port (`http://localhost:5601`)
 4. Install and configure [Elastic-Package](https://github.com/elastic/elastic-package)
 5. Set up the local env:
 
-```zsh
-just setup-env
-```
-
+- Install & activate hermit<br />
```

Comment:
replace <br /> with two spaces for new line

Reply:
Done